.svglintrc.js

```diff
@@ -46,6 +49,18 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            const iconPath = $.find("path").attr("d");
+            const bounds = getBounds(iconPath);
+            const width = +bounds.width.toFixed(floatPrecision);
+            const height = +bounds.height.toFixed(floatPrecision);
+
+            if (width === 0 && height === 0) {
+              reporter.error(`Path bounds reported as 0 x 0; check the path is valid`);
```

Comment:
Update the error message so it reads as an English sentence (and I'd suggest using `"` since this string is not actually a template literal)

```suggestion
              reporter.error("Path bounds reported as 0 x 0; check if the path is valid");
```

Reply:
I adjusted this a tiny bit further to make it more readable as an English sentence